app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt

```diff
@@ -374,135 +373,6 @@ class BindableAdapterTest {
     }
   }
 
-  @Test
-  fun testSingleTypeAdapter_setLifecycleOwnerTwice_throwsException() {
-    val exception = assertThrows(IllegalStateException::class) {
-      TestModule.testAdapterFactory =
-        { singleTypeAdapter, _ -> createSingleAdapterWithoutView(singleTypeAdapter) }
-    }
-
-    assertThat(exception).hasMessageThat().contains("lifecycle owner has already been bound")
-  }
-
-  @Test
-  fun testMultiTypeAdapter_setLifecycleOwnerTwice_throwsException() {
-
-    val exception = assertThrows(IllegalStateException::class) {
-      TestModule.testAdapterFactory =
-        { _, multiTypeFactory -> createMultiViewTypeNoDataBindingBindableAdapter(multiTypeFactory) }
-    }
-
-    assertThat(exception).hasMessageThat().contains("lifecycle owner has already been bound")
-  }
-
-  @Test
-  fun testSingleTypeAdapter_withLiveData_noLifecycleOwner_doesNotRebindLiveDataValues() {
-    // Set up the adapter to be used for this test.
-    TestModule.testAdapterFactory = { singleTypeFactory, _ ->
-      createSingleViewTypeWithDataBindingAndLiveDataAdapter(singleTypeFactory)
-    }
-
-    launch(BindableAdapterTestActivity::class.java).use { scenario ->
-      val itemLiveData = MutableLiveData<String>("initial")
-      scenario.onActivity { activity ->
-        val liveData = getRecyclerViewListLiveData(activity)
-        liveData.value = listOf(LiveDataModel(itemLiveData))
-      }
-      testCoroutineDispatchers.runCurrent()
-
-      itemLiveData.postValue("new value")
-      testCoroutineDispatchers.runCurrent()
-
-      // Verify that the bound data did not change despite the underlying live data changing.
-      onView(atPosition(recyclerViewId = R.id.test_recycler_view, position = 0)).check(
-        matches(
-          withText("initial")
-        )
-      )
-    }
-  }
-
-  @Test
-  fun testSingleTypeAdapter_withLiveData_withLifecycleOwner_rebindsLiveDataValues() {
```

Comment:
This & the multi version one below seem worth keeping as this is the behavior that should always be the case now (given that a lifecycle owner is always bound). These tests will help catch future regression for when people accidentally break the lifecycle owner part (which I think we want to make sure doesn't break given the benefits and promises of this PR).

Reply:
Okay, noted.